src/test/java/core/basesyntax/dao/FileServiceImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.storage.FruitStorage;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+public class FileServiceImplTest {
+    private static final String INPUT_FILE_NAME = "src/test/java/testFiles/inputReportTest";
+    private static final String RESULT_FILE_NAME = "src/test/java/testFiles/resultReportTest";

```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/20326858/1bb4a0e5-74f6-4498-85a7-0efc4f3bcf52)

Reply:
Only **java** files can be in **java** folder!